### DISCUSSION TITLE
How to make a separate role apart from admin like 'sub_admin' or 'developer' ?

### DISCUSSION THREAD

**Discussion by DellwinJohnso** [mozilla/hubs#4891] [Hubs Cloud Questions]
I have successfully deploy hubs cloud on my custom domain and 
My requirement is to make accounts with roles having controlled access in admin panel, such as account with  'developer' would have access to only import content, scenes and avatars. 
<img width="211" alt="image" src="https://user-images.githubusercontent.com/95520704/144701534-1ec6f32b-4cad-4c00-9a95-35fff28ae2c8.png">
 For example here, I would be having an option 'is_developer' and when the users(developers in this case) with this accounts login to the admin they should only see this import content, scenes and avatars.

Currently, we can find if the person is admin or not using this function, similarly I would like to fetch whether the logged in user is developer or not.
<img width="321" alt="image" src="https://user-images.githubusercontent.com/95520704/144701592-2a2e718c-6c3e-4b21-9ba2-668a0be701ae.png">

**Comment by pattersonbl2** [comment]
This question would be great to also ask in our community discord. You can join here (https://discord.com/invite/dFJncWwHun). I will ask my team if they may have a quick answer as well.
